src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -101,6 +107,34 @@ private void setUpEventHandler() {
             ui.triggerEvent(new PanelClickedEvent(this.panelIndex));
             requestFocus();
         });
+
+        ui.registerEvent((RepoOpeningEventHandler) e -> {
+            logger.info("Received RepoOpeningEvent");
```

Comment:
I'm not sure this is useful, since we already have logging for when events are triggered. Maybe something more high-level, like 'panel loading', 'panel loaded'?